src/Microsoft.Data/StringDataFrameColumn.BinaryOperations.cs

```diff
@@ -42,54 +42,74 @@ public override DataFrameColumn Add<T>(T value, bool inPlace = false)
             return ret;
         }
 
-        public override PrimitiveDataFrameColumn<bool> ElementwiseEquals(DataFrameColumn column)
+        internal static PrimitiveDataFrameColumn<bool> ElementwiseEqualsImplementation(DataFrameColumn left, DataFrameColumn column)
```

Comment:
Let me know if you want me to put this into its own PR. The test case I added for the IDataView->DataFrame unit test failed because this method was missing on `ArrowStringDataFrameColumn` and it was easy enough to refactor